### PR TITLE
Allow in-line editing in FC sets grid

### PR DESCRIPTION
**Pull request by DettRoxx**
### What does it do?
Adds ability to edit as inline fields in grid in FC sets as in FC Profiles grid

### Why is it needed?
Increase productivity

### Related issue(s)/PR(s)
#5225

### CODE REVIEW

File: core/model/modx/processors/security/forms/set/updatefromgrid.class.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        $data = $this->getProperty('data');
+        if (empty($data)) return $this->modx->lexicon('invalid_data');
+        $properties = $this->modx->fromJSON($data);
+        $this->setProperties($properties);
+        $this->unsetProperty('data');
+
+        return parent::initialize();
+    }
+}
+return 'modFormCustomizationSetUpdateFromGridProcessor';
```
Comment (alroniks) [inline core/model/modx/processors/security/forms/set/updatefromgrid.class.php:29]: Better to write `return modFormCustomizationSetUpdateFromGridProcessor::class` here.

**Comment by alroniks** [comment]
This RP will be on hold before Jason finish his refactoring.

**Review by Mark-H — APPROVED** [review]

**Comment by JoshuaLuckers** [comment]
@DettRoxx thanks for the patience, is it possible for you to refactor this PR?

**Comment by Mark-H** [comment]
The processor needs to be redone in the new way. If someone wants to pitch in here to help that would be awesome.

**Review by Mark-H — CHANGES_REQUESTED** [review]
Processor needs to be redone after the processor refactor.

**Comment by theboxer** [comment]
The processor already exists in the 3.x codebase, to make this work only the changed JS file is needed, with updated processor's path. (Tested and works fine)

**Comment by opengeek** [comment]
This was incorporated via #14781

**Pull request closed without merging**